charts/consul/templates/terminating-gateways-deployment.yaml

```diff
@@ -202,50 +203,21 @@ spec:
             - "-ec"
             - |
                 {{- if $root.Values.global.acls.manageSystemACLs }}
-                consul-k8s-control-plane acl-init \
-                  -component-name=terminating-gateway/{{ template "consul.fullname" $root }}-{{ .name }} \
-                  -acl-auth-method={{ template "consul.fullname" $root }}-k8s-component-auth-method \
-                  {{- if $root.Values.global.adminPartitions.enabled }}
-                  -partition={{ $root.Values.global.adminPartitions.name }} \
+                consul-k8s-control-plane connect-init -pod-name=${POD_NAME} -pod-namespace=${POD_NAMESPACE} \
+                  -consul-api-timeout={{ .Values.global.consulAPITimeout }} \
+                  -gateway -gateway-name="terminating-gateway" \
+                  -consul-node-name=${POD_NAMESPACE} \
```

Comment:
Node name should not be namespace.

Reply:
From RFC, Node name is the Consul Synthetic Node which in this case is "k8s-service-mesh".